manifest.json.tmpl

```diff
@@ -1,10 +1,10 @@
 {
     "manifest_version": 2,
     "name": "Tab Split 2",
-    "version": "0.1.0",
-    "description": "Open a mobile view of a page in the sidebar",
-    "author": "Ian Bicking",
-    "homepage_url": "https://github.com/mozilla/tab-split-2",
+    "version": "{{version}}",
+    "description": "{{description}}",
+    "author": "{{{author}}}",
+    "homepage_url": "{{{homepage}}}",
```

Comment:
These 4 values will get copied from package.json during `npm run build:manifest` (see below).
I didn't bother with anything else, since I figured most of the rest of these are unique to manifest.json, or are too confusing to normalize (ie: package.json's `name` versus manifest.json's completely different `name`).